pynxtools/dataconverter/readers/apm/utils/apm_create_nx_default_plots.py

```diff
@@ -31,73 +41,61 @@ def create_default_plot_reconstruction(template: dict, entry_id: int) -> dict:
     xyz = template[f"{trg}reconstructed_positions"]["compress"]
 
     print(f"\tEnter histogram computation, np.shape(xyz) {np.shape(xyz)}")
-    resolution = 1.0  # cubic voxel edge length in nm
-    bounds = np.zeros([3, 2], np.float32)  # in nm
-    for i in np.arange(0, 3):
-        bounds[i, 0] = np.min(xyz[:, i])
-        bounds[i, 1] = np.max(xyz[:, i])
-    # make the bounding box a quadric prism
-    imi = np.floor(bounds[0, 0]) - resolution
-    imx = np.ceil(bounds[0, 1]) + resolution
-    xedges = np.linspace(imi, imx, num=int(np.ceil((imx - imi) / resolution)) + 1,
-                         endpoint=True)
-    # this partitioning is not general enough, imi and imx should be left and right
-    # bounds respectively
-    imi = np.floor(bounds[1, 0]) - resolution
-    imx = np.ceil(bounds[1, 1]) + resolution
-    yedges = np.linspace(imi, imx, num=int(np.ceil((imx - imi) / resolution)) + 1,
-                         endpoint=True)
-    imi = np.floor(bounds[2, 0]) - resolution
-    imx = np.ceil(bounds[2, 1]) + resolution
-    zedges = np.linspace(imi, imx,
-                         num=int(np.ceil((imx - imi) / resolution)) + 1,
-                         endpoint=True)
+    # make the bounding box a quadric prism, discretized using cubic voxel edge in nm
+    aabb: dict = {"x": [0., 0.],
```

Comment:
Is it possible to find a better (i.e., more self-explanatory) name than `aabb`? Maybe not, I don't really know the use case.

Reply:
In general, variable names in this function could be improved.

Reply:
"aabb" is short-hand for axis_aligned bounding-box which is an accepted term in the computational geometry community. imi and imx indicate extrema values for dimensions where the i is a placeholder for x, y, z, admittedly maybe not pythonic enough

Reply:
Maybe add some comments here in the code to briefly explain these terms.